package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.0.5",
+  "version": "2.1.5",
```

Not sure if I should be updating minor version here. Because this commit is adding optional tag to textInput (a new thing and not an update). I think this is right but please verify.

---

A minor version bump is correct! Minor version bumps represent backwards-compatible feature additions. Patch version bumps (the last digit) represent backwards-compatible bug fixes. And major version bumps (the first digit) represent backwards-incompatible changes

---

If we do a minor version bump here, let's make sure to zero out the patch version as well 🙂 

```suggestion
  "version": "2.1.0",
```

https://semver.org/

---

^ Ah yes great catch!